pkg/cli/cli.go

```diff
@@ -132,40 +145,36 @@ func createOmittersFromConfig(config *schema.SchemaJson) (omitter.ReportingOmitt
 func createObfuscatorsFromConfig(config *schema.SchemaJson) (*obfuscator.MultiObfuscator, error) {
 	var obfuscators []obfuscator.ReportingObfuscator
 	for _, o := range config.Config.Obfuscate {
+		var (
+			k   obfuscator.ReportingObfuscator
+			err error
+		)
 		switch o.Type {
 		case schema.ObfuscateTypeKeywords:
-			k := obfuscator.NewKeywordsObfuscator(o.Replacement)
-			k = obfuscator.NewTargetObfuscator(o.Target, k)
-			obfuscators = append(obfuscators, k)
+			k = obfuscator.NewKeywordsObfuscator(o.Replacement, o.Report)
```

Comment:
as mentioned on slack, just create an already initialized tracker instead of passing the raw map around. You already did a great job here to refactor the loop, so you can just init at the top and pass down.

Reply:
Sure